### ISSUE TITLE
Can't seem to set ensimeIgnoreScalaMismatch

### ISSUE THREAD

**Issue by arigoldx** [ensime/ensime-sbt#293]
Hi there,

I read the help a few times and, after grokkin' this:

> Feel free to ask questions on the github issue tracker or gitter.im/ensime/ensime-server (there is no dedicated chat room for sbt-ensime).

I thought posting something here would be better 'cause gitter didn't seem like the place.

That said... should this be sufficient for a project-wide ensime.sbt file?

```
ensimeIgnoreScalaMismatch := true
```

There wasn't much info about `ensimeIgnoreScalaMismatch` online so I searched the github repo to see how it was used and found [a test](https://github.com/ensime/ensime-sbt/blob/e62cdb28f3b160f632b3ae4aa843726fa021814f/src/sbt-test/sbt-ensime/different-scalaorganization/build.sbt#L10) and copied that. Granted I'm new to futzing with sbt so maybe I'm just missing something simple.

For what it's worth, I can't post my project 'cause its, well, my company's proprietary stuff. That said, I'm committed to helpin' so if need be I'll make a faux project to try to reproduce my issue:

```
[info] ENSIME processing concept-definition (concept-definition)
[warn] You have a different version of scala for ENSIME (2.10.6) and concept-definition (2.11.8).
[warn] If this is not what you intended, try fixing your build with:
[warn]   scalaVersion in ThisBuild := "2.11.8"
[warn] in your build.sbt, or override the ENSIME scala version with:
[warn]   ensimeScalaVersion in ThisBuild := "2.11.8"
[warn] in a ensime.sbt: http://ensime.org/build_tools/sbt/#customise
org.ensime.EnsimePlugin$$anon$1: To ignore this error (i.e. you have multiple scala versions), customise `ensimeIgnoreScalaMismatch`
[error] org.ensime.EnsimePlugin$$anon$1: To ignore this error (i.e. you have multiple scala versions), customise `ensimeIgnoreScalaMismatch`
[error] Use 'last' for the full log.
```

basically we have a bunch of sub-projects which use different versions of Scala -- when I set `ensimeScalaVersion in ThisBuild := "2.11.8"` it failed on the next project which wanted `2.10.6`.

Thanks a pile! Y'all superstars!

**Comment by fommil** [comment]
Yeah sure, feel free to ignore the explicit advice we've given, that makes our lives easier. Gitter is for support, github is for actual bugs.

**Comment by fommil** [comment]
Fwiw. It's probably something like `ensimeThing in LocalProject("foo") := true`

Please use gitter next time.

**Comment by arigoldx** [comment]
A thousand pardons.. I really did try to cover all my bases. 'suppose I was confused by:

> Feel free to ask questions on the github issue tracker or gitter.im/ensime/ensime-server

I use GH a lot but maybe I misunderstood what you meant by the "github issue tracker"... it won't happen again.

**Comment by fommil** [comment]
A pr to update the docs world be appreciated. Support is gitter only, and purely volunteer basis.

**Comment by arigoldx** [comment]
[Done](https://github.com/ensime/ensime.github.io/pull/196).
